dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/controller/ExecutorController.java

```diff
@@ -138,6 +144,98 @@ public Result startProcessInstance(@ApiIgnore @RequestAttribute(value = Constant
         return returnDataList(result);
     }
 
+    /**
+     * batch execute process instance
```

Comment:
Please also add https://github.com/apache/dolphinscheduler/pull/7771/files#r778501392 to comment, thanks

Reply:
BTW, please rebase to dev branch, we fix flaky test in our CI